### DISCUSSION TITLE
Is there a way to pick neighbors?

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7652] [Q&A] [answered]
Hello,

I have 10 full-thread devices and 1 border router, and I want the data to hop to specific devices of my choice before reaching the border router. Is that possible?

**Comment by jwhui** [comment, marked as answer]
Thread does not provide a standard way to control or constrain the topology.

OpenThread does provide a [macfilter](https://github.com/openthread/openthread/tree/main/src/cli#macfilter) feature that we often use in testing. The macfilter feature provides filtering rules that can limit what neighbors a given device receives from.

Hope that helps.

**Reply by simonlingoogle** [reply]
> OpenThread does provide a [macfilter](https://github.com/openthread/openthread/tree/main/src/cli#macfilter) feature that we often use in testing. The macfilter feature provides filtering rules that can limit what neighbors a given device receives from.

The border router can be restricted by `macfilter` to only be able to connect with the specific device. That's a simple way to achieve what @kmfnjtze8888 wants.
